content/rest-api/filter.md

```diff
@@ -576,3 +576,23 @@ You can filter the reference entity records to get only the completed ones on a
 /api/rest/v1/reference-entities/brands/records?search={"complete":[{"operator":"=","value":true,"scope":"ecommerce","locales":["en_US"]}]}
 ```
 
+### Filter records by update date
+
+You can filter the reference entity records by their update date.
+
+Below are the allowed operators to filter on this property, as well as the corresponding type of value expected in the `search` query parameter.
+
+:::info
+Note that dates are interpreted in the time zone of the server that runs Akeneo PIM (e.g. date.timezone setting in php.ini).
```

Comment:
Will we be able to filter with a time-zoned time? Or is it too complex? @ahocquard

Reply:
Done!